src/composables/QuasarFieldRule.js

```diff
@@ -20,3 +20,14 @@ export function isGitRepositoryUrl(t, value) {
   return /^http(s)?:\/\/.+\/.*(?<!\.git)$/.test(value)
     || t('errors.invalid.gitProvider.repository');
 }
+/**
+ * Check if branch name is not already exists.
+ * @param {Function} t - I18n translate function.
+ * @param {String} value - Value to check.
+ * @return {boolean|String} Return true if the branch not already exists otherwise the translated
+ * error message.
+ */
+export function isDuplicateBranchName(t, branches, value) {
```

Comment:
function name is confusing => if it returns true, it seems that the branch is indeed duplicated but the description of the return says the opposite. Why not renaming `canCreateBranch` or `doesBranchNameAlreadyExist` or else

Reply:
I will improve the documentation :)